includes/functions/content-sync.php

```diff
@@ -269,7 +269,7 @@ function get_post_data( $post ) {
 		'publishedAt'         => gmdate( \DateTime::RFC3339, strtotime( $post->post_date_gmt ) ),
 		'plainText'           => wp_strip_all_tags( $content ),
 		'size'                => str_word_count( wp_strip_all_tags( $content ) ),
-		'sectionNames'        => Utils\get_post_categories( $post->ID ),
+		'allSections'         => Utils\get_post_categories( $post->ID ),
```

Comment:
Is this new field sending the category data across in the following requested sample format showing multiple, nested categories?:
> ['/world/asia-pacific','/business/healthcare-pharmaceuticals','/world']